core/kazoo_attachments/src/kz_att_s3.erl

```diff
@@ -65,3 +61,34 @@ convert_kv({K, V})
 convert_kv({<<"etag">> = K, V}) ->
     {K, binary:replace(V, <<$">>, <<>>, ['global'])};
 convert_kv(KV) -> KV.
+
+-spec get_map_values(kz_data:connection()) -> {ne_binary(), ne_binary(), ne_binary(), api_binary(), ne_binary()}.
+get_map_values(#{bucket := Bucket
+                ,key := Key
+                ,secret := Secret
+                ,path := Path
+                ,host := Host
+                }) ->
+    {Bucket, Key, Secret, Path, Host};
```

Comment:
simplify this matching only bucket, key, secret and use `maps:get(.., Map, Default) for path and host